app/application/route.js

```diff
@@ -0,0 +1,45 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class ApplicationRoute extends Route {
+  @service store;
+
+  beforeModel() {
+    const request1 = this.store.createRecord('request', {
+      id: 1,
+      type: 'breakdown',
+      requestor: 'user1',
+      responder: 'user4',
```

Comment:
When you fix attributes in `request` model then these keys will expect user `Model` instead `string` value